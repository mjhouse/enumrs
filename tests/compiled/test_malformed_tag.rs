use enumrs::Tagged;

#[derive(Tagged)]
pub enum TestEnum {
    // Tags that have expressions must be delimited
    // with a comma.
    #[tag(tagname 10)]
    Variant = 1,
}

pub fn main() {}