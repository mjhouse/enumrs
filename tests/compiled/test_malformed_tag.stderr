error: Malformed tag: 'tagname 10'
 --> tests/compiled/test_malformed_tag.rs:7:11
  |
7 |     #[tag(tagname 10)]
  |           ^^^^^^^
